app/services/search_instrumentation_service.rb

```diff
@@ -0,0 +1,52 @@
+class SearchInstrumentationService
+  def self.log_search_suggestions_results(query, results)
+    results_count = results[:data].size
+    results_zero = results_count.nil? || results_count.zero?
+    query_length = query.present? ? query.length : 0
+    Rails.logger.info "Search Suggestion Request : search_query=#{query}, query_length=#{query_length}, "\
```

Comment:
I’m not sure these are queryable properties?

Reply:
Changed to json format